docs/user_guide/datasets/acs_income.rst

```diff
@@ -0,0 +1,171 @@
+years old who worked at least 1 hour per week in the past year and had an 
+income of at least $100.
+
+
+.. _acsincome_dataset_description:
+
+Dataset Description
+^^^^^^^^^^^^^^^^^^^
+Ding et al. provide data from 2014-2018 for all 50 states and Puerto Rico.
+Note that Puerto Rico is the only US territory included in this dataset.
+We uploaded the 2018 data to OpenML, which anyone can access.
+The dataset contains 1,664,500 rows. Each row describes a person and contains 
+10 features, which we describe below:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 7 30
+   :stub-columns: 1
+
+   *  - Column name
+      - Description
+
+   *  - AGEP
+      - Age as an integer from 0 to 99
+
+   *  - COW
+      - Class of worker:
+         1. Employee of a private for-profit company or business, or of an individual, for wages, salary, or commissions 
+         2. Employee of a private not-for-profit, tax-exempt, or charitable organization 
+         3. Local government employee (city, county, etc.) 
+         4. State government employee 
+         5. Federal government employee 
+         6. Self-employed in own not incorporated business, professional practice, or farm 
+         7. Self-employed in own incorporated business, professional practice or farm 
+         8. Working without pay in family business or farm 
+         9. Unemployed and last worked 5 years ago or earlier or never worked
+
+   *  - SCHL
+      - Educational attainment:
+         1. No schooling completed
+         2. Nursery school or preschool
+         3. Kindergarten
+         4. Grade 1
+         5. Grade 2
+         6. Grade 3
+         7. Grade 4
+         8. Grade 5
+         9. Grade 6
+         10. Grade 7
+         11. Grade 8
+         12. Grade 9
+         13. Grade 10
+         14. Grade 11
+         15. Grade 12 (no diploma)
+         16. Regular high school diploma
+         17. GED or alternative credential
+         18. Some college but less than 1 year
+         19. 1 or more years of college credit but no degree
+         20. Associate's degree
+         21. Bachelor's degree
+         22. Master's degree
+         23. Professional degree beyond a bachelor's degree
+         24. Doctorate degree
+
+   *  - MAR
+      - Marital status:
+         1. Married
+         2. Widowed
+         3. Divorced
+         4. Separated
+         5. Never married or under 15 years old
+
+   *  - OCCP
+      - Occupation. There are over 500 categories. Please see data dictionary at `ACS PUMS documentation <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_ for the full list of occupation codes.
+
+   *  - POBP
+      - Place of birth. There are over 200 categories, including the 50 US states and several countries. Please see the data dictionary at `ACS PUMS documentation <https://www.census.gov/programs-surveys/acs/microdata/documentation.2018.html>`_ for the full list.
+
+   *  - RELP
+      - Relationship:
```

Comment:
I think this feature could use a bit more explanation. Relationship to whom? The reference person?

Reply:
Thanks for this question. Page 41 of 2018 US Census Data Dictionary ([link](https://www2.census.gov/programs-surveys/acs/tech_docs/pums/data_dict/PUMS_Data_Dictionary_2018.pdf)) and page 20 of the paper's SI ([link](https://arxiv.org/pdf/2108.04884.pdf)) only say relationship with no other information provided. However, I found this separate appendix from the US Census ([link](https://www.census.gov/content/dam/Census/library/publications/2020/acs/acs_pums_handbook_2020_appendix_a.pdf)) that says this is "relationship to householder". I'll update the description.